espanol/revolucion/revolucion.md

```diff
@@ -1 +1,3 @@
-Indicando que sus planes revolucionarios deberian tener un espacio y tiempo adecuado para su realizacion.
\ No newline at end of file
+Indicando que sus planes revolucionarios deberian tener un espacio y tiempo adecuado para su realizacion.
+
+Sabiendo que era su ejercito el que se acercaba, la preocupacion de como [alimentar a un ejercito](alimentar_ejercito/como_alimentar_un_ejercito.md) se apodera de todo su ser.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.